manifests/repo.pp

```diff
@@ -87,7 +87,7 @@
           }
         }
 
-        if ($facts['os']['release']['major'] == '7' and versioncmp($zabbix_version, '5.0') >= 0) {
+        if ($facts['os']['name'] == 'CentOS' and $facts['os']['release']['major'] == '7' and versioncmp($zabbix_version, '5.0') >= 0) {
```

Comment:
Will this also affect RHEL? If you're running RHEL, you probably don't want CentOS SCL bits, but it can break some environments.

Reply:
Oh yes, with Foreman we had copied it in our repos so it worked for other OSes (IIRC mostly for Scientific Linux).

Reply:
Those were the times…